certbot.wrapper

```diff
@@ -35,4 +35,4 @@ export CERTBOT_AUGEAS_PATH="${SNAP}/usr/lib/${ARCH_TRIPLET}/libaugeas.so.0"
 CERTBOT_PLUGIN_PATH="$(curl -s --unix-socket /run/snapd.socket "http://localhost/v2/connections?snap=certbot&interface=content" | jq -r  '.result.established | map(select(.plug.plug == "plugin" and ."plug-attrs".content == "certbot-1") | "/snap/"+.slot.snap+"/current/lib/python3.8/site-packages/" ) | join(":")')"
 export CERTBOT_PLUGIN_PATH
 
-exec certbot "$@"
+exec certbot --preconfigured-renewal "$@"
```

Comment:
This has an annoying side-effect, something to do with the argparse library we use. it breaks:

    $ sudo certbot help
    usage:
      certbot [SUBCOMMAND] [options] [-d DOMAIN] [-d DOMAIN] ...

    Certbot can obtain and install HTTPS/TLS/SSL certificates.  By default,
    it will attempt to use a webserver both for obtaining and installing the
    certificate.
    certbot: error: unrecognized arguments: help

Perhaps that is not an officially supported subcommand (as opposed to `certbot --help`), but maybe someone is relying on it.

A possible workaround is to postfix `--preconfigured-renewal` rather than prefixing it, but I'm even less sure of what parser dragons might be hiding there.

To the best of my knowledge, only `help` is affected, but I haven't rigorously tried every command (outside of tests).

Reply:
To relate to a similar situation, on integration tests a default command is used for certbot that include a lot of flags:
https://github.com/certbot/certbot/blob/3a615176c5edae3a6a891961d0ae577e42b69a16/certbot-ci/certbot_integration_tests/utils/certbot_call.py#L89-L105

Then flags are postfixed to it to match specific tests requirements. For instance:
https://github.com/certbot/certbot/blob/3a615176c5edae3a6a891961d0ae577e42b69a16/certbot-ci/certbot_integration_tests/certbot_tests/test_main.py#L106-L114

And everything worked fine so far. So I think it is quite safe to postfix the flag instead of prefixing it.

Reply:
OK, that's reassuring, thanks Adrien. Postfix it is.

Another smoke test: with `certbot register -m`, the unfinished `-m` doesn't eat the `--preconfigured-renewal`. Hopefully that means the postfixed flag's behavior won't/can't be influenced by preceding arguments, in the general case.